rust/src/shards/fs.rs

```diff
@@ -135,6 +148,13 @@ impl BlockingShard for FileDialog {
         dialog = dialog.add_filter(filter, &[filter]);
       }
     }
+    let cd = self.cd.get();
```

Comment:
what's the default? process current?

Reply:
It seems to depend on the OS.
On Windows it remembers latest location. On my Ubuntu it goes to "recent locations". It's likely driven by the OS itself.